manticore/platforms/evm.py

```diff
@@ -2058,18 +2086,23 @@ def send_funds(self, sender, recipient, value):
 
     def get_code(self, address):
         if address not in self._world_state:
-            return bytearray()
+            return bytes()
```

Comment:
What's the reason of that change?

Reply:
Ok I guess its to be able to specify code with `b''`

Reply:
bytecode is generally not mutable also. bytearray sounded like a bad fit